core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala

```diff
@@ -130,6 +132,10 @@ class DruidBeamMaker[A](
         log.warn(s"DruidTuning key[$k] for task[$taskId] overridden from[$v] to[${druidTuningMapWithOverrides(k)}].")
       }
     }
+    val taskContextMap = taskContext match {
+      case taskContext if taskContext.nonEmpty => Map("context" -> taskContext)
```

Comment:
Using the same name in pattern matching as original value (on which we do pattern matching) is suspicious and in some cases might lead to errors.